README.md

```diff
@@ -25,8 +25,7 @@ The Intelligent Kiosk Sample is a collection of demos showcasing workflows and e
 | [Bing Visual Search](Documentation/BingVisualSearch.md)      | Shows how to use the power of Bing Image Insights to perform visual search and find visually similar images, products or celebrities. | Bing Search API;  Bing AutoSuggestion API |
 | [Custom Vision Explorer](Documentation/CustomVisionExplorer.md)      | Shows how to use the Custom Vision Service to create a custom image classifier or object detector and score images against it. | Custom Vision API, Bing Image Search API;  Bing AutoSuggestion API |
 | [Custom Vision Setup](Documentation/CustomVisionSetup.md)      | Shows how to train a machine learning model using the Custom Vision Service. | Custom Vision API, Bing Image Search API;  Bing AutoSuggestion API |
-| [Emotion API Explorer](Documentation/EmotionAPIPlayground.md)       | A playground for the Emotion APIs | Windows 10 Face Tracking; Emotion prediction; Bing Search API; Bing AutoSuggestion API |
-| [Face API Explorer](Documentation/FaceAPIPlayground.md)          | A playground for the Face APIs used for age and gender prediction, as well as face identification | Windows 10 Face Tracking; Age and gender prediction; Face identification; Facial landmarks; Bing Image Search API; Bing AutoSuggestion API |
+| [Face API Explorer](Documentation/FaceAPIExplorer.md)          | A playground for the Face APIs used for extraction a series of face-related attributes, such as head pose, gender, age, emotion, facial hair, and glasses, as well as face identification. | Windows 10 Face Tracking; Face API; Bing Image Search API; Bing AutoSuggestion API |
```

Comment:
... used for extracting face-related attributes ...

Reply:
fixed